docs/dataset/quickstart.md

```diff
@@ -5,3 +5,185 @@ icon: kolena/flame-16
 # :kolena-flame-20: Quickstart
 
 Install Kolena to set up rigorous and repeatable model testing in minutes.
+
+In this quickstart guide, we'll use we will use the [300 Faces In the Wild (300-W)](https://ibug.doc.ic.ac.uk/resources/300-W/)
+example dataset to demonstrate the how to curate test data and test models in Kolena. The examples
+use data located in a public S3 bucket `s3://kolena-public-examples`
+
+## Installing `kolena` Python SDK
+
+=== "`pip`"
+
+    ```shell
+    pip install kolena
+    ```
+
+=== "`poetry`"
+
+    ```shell
+    poetry add kolena
+    ```
+
+## Clone the examples
+
```

Comment:
I'd move this into a callout — most users will do the easiest thing (not cloning our repo) and most readers will skip over callouts